.eslintrc.json

```diff
@@ -23,6 +23,7 @@
   "ignorePatterns": "dist/",
   "plugins": ["vue", "@typescript-eslint"],
   "rules": {
-    "@typescript-eslint/ban-ts-comment": ["error", { "ts-ignore": "allow-with-description" }]
+    "@typescript-eslint/ban-ts-comment": ["error", { "ts-ignore": "allow-with-description" }],
+    "vue/no-setup-props-destructure": "off"
```

Comment:
I'm not sure props destructure is a good thing. Eslint suggest that will lose reactivity after destructure.

Reply:
The behavior is changed in a recent update. https://github.com/vuejs/rfcs/discussions/369

Reply:
Now I finally know why this rfc makes so many arguements. It is easy to use but needs developer to learn.

I think this pr is ok to merge. I may add some comments after merged to avoid future miss usage.